src/Sulu/Bundle/ContactBundle/Contact/ContactManagerInterface.php

```diff
@@ -40,21 +43,21 @@ public function removeAddressRelation($entity, $address);
     /**
      * Returns a collection of relations to get addresses.
      *
-     * @param string $entity
+     * @param ContactInterface $entity
      */
     public function getAddressRelations($entity);
 
     /**
      * sets the first element to main, if none is set.
      *
-     * @param array $arrayCollection
+     * @param ArrayCollection $arrayCollection
```

Comment:
same as for for the other $arrayCollection this should be typehinted with:

```suggestion
     * @param Collection $arrayCollection
```